README.md

```diff
@@ -47,3 +47,24 @@ We use pre-commit to make sure the code is consistently formatted. To make sure
 ### Package Build
 
 To build the python package: `$ poetry build`
+
+## Release Instructions
+
+The following instructions apply when doing release manually.
+
+* Check and update the version in "pyproject.toml" file.
+* Check and update the version in "custom-nb-image/VERSION" file.
+* Update the codeflare-sdk version in "custom-nb-image/Dockerfile".
+* Commit the changes of "pyproject.toml" and "VERSION" to the repository.
```

Comment:
Still need to commit the change to `custom-nb-image/Dockerfile` after adding it. Also, I think that updating VERSION and Dockerfile should probably just happen after the package release (as updating those before release will cause automation to kick off and try to build new NB image, but fail because new package isn't released). It may be best to split the instructions into:
 - Package release
 - Notebook image release